app/src/main/java/org/oppia/app/topic/reviewcard/ReviewCardFragmentPresenter.kt

```diff
@@ -0,0 +1,49 @@
+package org.oppia.app.topic.reviewcard
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.oppia.app.R
+import org.oppia.app.databinding.ReviewCardFragmentBinding
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.viewmodel.ViewModelProvider
+import javax.inject.Inject
+
+/** Presenter for [ConceptCardFragment], sets up bindings from ViewModel */
+@FragmentScope
+class ReviewCardFragmentPresenter @Inject constructor(
+  private val fragment: Fragment,
+  private val viewModelProvider: ViewModelProvider<ReviewCardViewModel>
+) {
+  private lateinit var subtopicId: String
+  private lateinit var topicId: String
```

Comment:
Remove these two, you can directly us topicId and subtopicId in handleCreateView

Reply:
Done.